context-monitoring/src/main/java/de/atb/context/persistence/monitoring/MonitoringDataRepository.java

```diff
@@ -57,7 +50,7 @@
  * @author scholze
  * @version $LastChangedRevision: 143 $
  */
-public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositorySDB<Type> implements
+public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositoryTDB<Type> implements
```

Comment:
There are a few `protected` methods in this class but that does not make much sense if the class is `final` and cannot be extended.

Reply:
[x] done